app/uk/gov/hmrc/emcstfe/models/response/SubmitDraftMovementResponse.scala

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.emcstfe.models.response
+
+import cats.implicits.catsSyntaxTuple2Semigroupal
+import com.google.common.io.BaseEncoding
+import com.lucidchart.open.xtract.{XPath, XmlReader, __}
+import play.api.libs.json.{Json, OWrites}
+
+import java.nio.charset.StandardCharsets
+import java.util.Base64
+
+case class SubmitDraftMovementResponse(
+                                        receipt: String,
+                                        lrn: Option[String] = None
+                                      )
+
+object SubmitDraftMovementResponse {
+
+  private[response] def digestValueToReceipt(dv: String): String = {
```

Comment:
We need to be able to convert from ChRIS's encoded value to a more "useful" one. This function takes a Base64 encoded value, decodes it, then re-encodes it as Base32. This replicates what the existing service does (except it uses Java/Google libraries rather than homebrewing its own solution).

Example input: `KWrqNXggsCEMgbOr3wiozyfrdU0=`
Example output: `FFVOUNLYECYCCDEBWOV56CFIZ4T6W5KN`